CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Not released (2018-10-29)
 -------------------------
 
 Bugs fixed:
+ * relaxed bundler dependencies (Keiji, Yoshimi)
```

Comment:
The changelog is autogenerated from commit messages, see CONTRIBUTING.md.

Reply:
Oh. I see.

Reply:
@walf443 Could you revert this change please?

Reply:
@iliabylich I'm sorry to reply slowly. I fixed it.